source/tutorials/longitudinal.rst

```diff
@@ -114,5 +114,61 @@ Second, control charts display the mean value of "metric" at each "state". The f
+
+First let's download a feature table to test.
+
+.. download::
+   :url: https://data.qiime2.org/2017.9/tutorials/longitudinal/ecam_table_taxa.qza
+   :saveas: ecam-table-taxa.qza
+
+Now we are ready run NMIT. The output of this command is a distance matrix that we can pass to other QIIME2 commands for significance testing and visualization.
+
+.. command-block::
+
+   qiime longitudinal nmit \
+     --i-table ecam-table-taxa.qza \
+     --m-metadata-file ecam_map_maturity.txt \
+     --p-individual-id-column studyid \
+     --o-distance-matrix nmit-dm.qza
+
+
+Now let's put that distance matrix to work. First we will perform PERMANOVA tests to evaluate whether between-group distances are larger than within-group distance.
+
+.. note:: NMIT computes between-subject distances across all time points, so each subject (as defined the ``--p-individual-id-column`` parameter used above) gets compressed into a single "sample" representing that subject's longitudinal microbial interdependence. This new "sample" will be labeled with the ``SampleID`` of one of the subjects with a matching ``individual-id``; this is done for the convenience of passing this distance matrix to downstream steps without needing to generate a new sample metadata file but it means that you must **pay attention**. **For significance testing and visualization, only use group categories that are uniform across each ``individual-id``. DO NOT ATTEMPT TO USE METADATA CATEGORIES THAT VARY OVER TIME OR BAD THINGS WILL HAPPEN.** For example, in the tutorial metadata a patient is labeled ``antiexposedall==y`` only after antibiotics have been used; this is a category that you should not use, as it varies over time. Now have fun and be responsible.
+
+.. command-block::
+
+   qiime diversity beta-group-significance \
+     --i-distance-matrix nmit-dm.qza \
+     --m-metadata-file ecam_map_maturity.txt \
+     --m-metadata-category delivery \
+     --o-visualization nmit.qzv
+
+Finally, we can compute principal coordinates and use Emperor to visualize similarities among **subjects** (not individual samples; see the note above). 
+
+.. command-block::
+
+   qiime diversity pcoa \
+     --i-distance-matrix nmit-dm.qza \
+     --o-pcoa nmit-pc.qza
+
+.. command-block::
+
+   qiime emperor plot \
+     --i-pcoa nmit-pc.qza \
+     --m-metadata-file ecam_map_maturity.txt \
+     --o-visualization nmit-emperor.qzv
+
+So there it is. We can use PERMANOVA test or other distance-based statistical tests to determine whether groups exhibit different longitudinal microbial interdependence relationships, and PCoA/emperor to visualize the relationships among groups of subjects. **Don't forget the caveats mentioned above about using and interpreting NMIT**. Now be safe and have fun.
+
 .. _ECAM study: https://doi.org/10.1126/scitranslmed.aad7121
 .. _statsmodels LME description page: http://www.statsmodels.org/dev/mixed_linear.html
+.. _Zhang et al., 2015: https://doi.org/10.1002/gepi.22065
```

Comment:
The reference year should be "2017"

Reply:
👍 thanks for catching this @elong0527 ! I have fixed this now.